pcomfortcloud/__main__.py

```diff
@@ -197,6 +202,27 @@ def main():
         help='date of day like 20190807')
 
     args = parser.parse_args()
+    
+    # if username has been specified as the string "auth" then password is actually a filename containing both the username and password
+    if args.username == "authfile":
```

Comment:
it would be better if username and password could be left out from the cli command instead of using a fake user and password.

If it isn't possible, maybe we should up the version and make a breaking change

Reply:
@majurgens If Mr. Authfile comes and tries this package, he will not be able to login without using authfile. I think we should use `#` or something like that, instead of the keyword `authfile`.

Reply:
That's a good idea, even if its highly unlikely for anyone to be called authfile :)
PR updated